src/commands/update.js

```diff
@@ -13,9 +13,10 @@ function brew (...args) {
   return cp.spawnSync('brew', args, {stdio: 'inherit'})
 }
 
+let cli = global.config ? global.config.bin : 'Heroku'
```

Comment:
I know this line previously contained `Heroku` instead of `heroku` but isn't the value of `config.bin` usually the lower case variant?  It doesn't matter to me per se, but the fallback might be more consistent with the expected pjson value if it were `heroku`?

Reply:
yeah let's just change the fallback for now, a real solution might be bringing in something like https://github.com/dreamerslab/node.inflection but that can happen another time